### ISSUE TITLE
Support for local file directories for view definition catalog

### ISSUE THREAD

**Issue by kahboom** [KaotoIO/kaoto-backend#65]
**Describe the solution you'd like**
I would like to be able to specify in `application.yaml` a file path like this:

```
repository:
  step:
    jar:
      - "https://repo1.maven.org/maven2/org/apache/camel/kamelets/camel-kamelets/0.6.0/camel-kamelets-0.6.0.jar"
      - "https://github.com/KaotoIO/camel-component-metadata/archive/refs/heads/main.zip"
  viewdefinition:
    git:
      -
        url: "file://<path to dir>/custom-viewdefinition-catalog"
```

**Describe alternatives you've considered**
Using the github repo path.

**Additional context**
This is convenient for testing and if, for example, you want to experiment with developing your view definition catalog locally.

**Comment by kahboom** [comment]
Don't we need something like this for VSCode anyway?

**Comment by Delawen** [comment]
This is already implemented in #89
